### PR TITLE
test: M3-8148: Cypress tests for refactored Linode Create flow with add ons

### PR DESCRIPTION
**Pull request by cliu-akamai**
## Description 📝
Add integration test to the flow of creating Linode with add-ons

## Major Changes 🔄
- Check users can create Linode with add-ons.

## How to test 🧪
```
yarn cy:run -s "cypress/e2e/core/linodes/create-linode-with-add-ons.spec.ts"
```

### CODE REVIEW

**Review by bnussman-akamai — APPROVED** [review]

**Review by dwiley-akamai — APPROVED** [review]
Re-ran `Code Coverage / current_branch (pull_request)` and it passed ✅
`create-linode-with-add-ons.spec.ts` passed in CI ✅ (local failures due to timeouts)

Should we have a changeset for this?

File: packages/manager/cypress/support/ui/pages/linode-create-page.ts [hunk 1]
```diff
@@ -91,4 +91,39 @@ export const linodeCreatePage = {
...
+    cy.findByText('Private IP').should('be.visible').closest('label').click();
+  },
+
+  /**
+   * Checks the EU agreements.
+   */
+  checkEUAgreements: () => {
+    cy.get('body').then(($body) => {
+      if ($body.find('div[data-testid="eu-agreement-checkbox"]').length > 0) {
+        // eslint-disable-next-line cypress/unsafe-to-chain-command
```
Comment (mjac0bs) [inline packages/manager/cypress/support/ui/pages/linode-create-page.ts:115]: @cliu-akamai - can you make sure you have our [linting extensions](https://linode.github.io/manager/development-guide/13-coding-standards.html#linting) installed in your editor? Seeing some linting warnings here that will be flagged and can be resolved easily with the ESLint extension.

Screenshot 2024-08-07 at 9 28 50 AM

**Review by mjac0bs — APPROVED** [review]
Thanks for the changeset. The CI failure looked like unit test flake (should resolve since that job has been restarted) and a Cypress failure on `stackscripts/smoke-community-stackscrips.spec.ts`, which is unrelated to this PR and [fixed in develop now](https://github.com/linode/manager/pull/10757).

File: packages/manager/cypress/support/ui/pages/linode-create-page.ts (hunk 1 above)
Comment (cliu-akamai) [reply packages/manager/cypress/support/ui/pages/linode-create-page.ts:115]: Sure

**Pull request merged**